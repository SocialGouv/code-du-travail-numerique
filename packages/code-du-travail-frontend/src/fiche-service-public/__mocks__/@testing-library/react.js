import React from "react";
import PropTypes from "prop-types";
import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import { theme } from "@socialgouv/cdtn-ui";

const AllTheProviders = ({ children }) => {
  return (
    <ThemeProvider theme={theme.colors}>
      <>{children}</>
    </ThemeProvider>
  );
};

AllTheProviders.propTypes = {
  children: PropTypes.node.isRequired,
};

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
