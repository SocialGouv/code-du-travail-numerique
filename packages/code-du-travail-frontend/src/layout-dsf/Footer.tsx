import { Footer as DsfrFooter } from "@codegouvfr/react-dsfr/Footer";
import React from "react";

const Footer = (): JSX.Element => {
  return (
    <DsfrFooter
      accessibility="fully compliant"
      contentDescription="
    Ce message est à remplacer par les informations de votre site.

    Comme exemple de contenu, vous pouvez indiquer les informations
    suivantes : Le site officiel d’information administrative pour les entreprises.
    Retrouvez toutes les informations et démarches administratives nécessaires à la création,
    à la gestion et au développement de votre entreprise.
    "
      termsLinkProps={{
        href: "#",
      }}
      websiteMapLinkProps={{
        href: "#",
      }}
    />
  );
};

export default Footer;
