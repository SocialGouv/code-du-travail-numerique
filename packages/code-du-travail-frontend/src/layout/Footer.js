import {
  Button,
  Container,
  FlatList,
  Heading,
  icons,
  Section,
  theme,
  Title,
} from "@socialgouv/cdtn-ui";
import getConfig from "next/config";
import Link from "next/link";
import { useRouter } from "next/router";
import React from "react";
import styled from "styled-components";

import { ContactModal } from "../common/ContactModal";
import { ServiceRenseignementModal } from "../common/ServiceRenseignementModal";
import { Partners } from "../home/Partners";
import { matopush } from "../piwik";

const { DirectionRight: DirectionRightIcon } = icons;
const { publicRuntimeConfig } = getConfig();

const GITHUB_REPO = "https://github.com/SocialGouv/code-du-travail-numerique";

const Footer = () => {
  const router = useRouter();
  const path = router.asPath;

  return (
    <OverflowWrapper>
      <StyledFooter role="contentinfo">
        <ServiceSection>
          <Container>
            <Title as="strong" isFirst stripe="top" isHeaderLess>
              Besoin de plus d’informations&nbsp;?
            </Title>
            <StyledContainer narrow noPadding>
              Les services du ministère du Travail en région informent,
              conseillent et orientent les salariés et les employeurs du secteur
              privé sur leurs questions en droit du travail.
            </StyledContainer>
            <ServiceRenseignementModal>
              {(openModal) => (
                <Button onClick={openModal}>
                  Contacter nos services en région
                  <StyledDirectionRightIcon />
                </Button>
              )}
            </ServiceRenseignementModal>
          </Container>
        </ServiceSection>

        {router.pathname === "/" && <Partners />}

        <NavSection>
          <Links>
            <Category>
              <Heading as={StyledStrong} isFirst>
                Code du travail numérique
              </Heading>
              <StyledList>
                <StyledListItem>
                  <Link passHref href="/droit-du-travail">
                    <StyledLink>Le droit du travail</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/glossaire">
                    <StyledLink>Glossaire</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/a-propos">
                    <StyledLink>À propos</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/mentions-legales">
                    <StyledLink>Mentions légales</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/accessibilite">
                    <StyledLink>
                      Accessibilité&nbsp;: partiellement conforme
                    </StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/politique-confidentialite">
                    <StyledLink>Politique de confidentialité</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/stats">
                    <StyledLink>Statistiques d’utilisation</StyledLink>
                  </Link>
                </StyledListItem>
                <StyledListItem>
                  <Link passHref href="/integration">
                    <StyledLink>
                      Intégrer le Code du travail numérique
                    </StyledLink>
                  </Link>
                </StyledListItem>
              </StyledList>
            </Category>
            <Category>
              <Heading as={StyledStrong} isFirst>
                Aidez-nous à améliorer cet outil
              </Heading>
              <StyledList>
                <StyledListItem>
                  <ContactModal>
                    {(openModal) => (
                      <Button
                        variant="navLink"
                        onClick={() => {
                          matopush([
                            "trackEvent",
                            "contact",
                            "click_contact_cdtn_team",
                            path,
                          ]);
                          openModal();
                        }}
                      >
                        Contact
                      </Button>
                    )}
                  </ContactModal>
                </StyledListItem>
                <StyledListItem>
                  <StyledLink
                    href={`${GITHUB_REPO}/tree/${publicRuntimeConfig.PACKAGE_VERSION}`}
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    Contribuer sur Github
                  </StyledLink>
                </StyledListItem>
                <StyledListItem>
                  {(() => {
                    const packageVersion =
                      publicRuntimeConfig.PACKAGE_VERSION || "";
                    const isTag = packageVersion[0] === "v";
                    const path = isTag
                      ? "releases/tag"
                      : packageVersion === "master"
                      ? "commits"
                      : "compare";
                    return (
                      <StyledLink
                        href={`${GITHUB_REPO}/${path}/${packageVersion}`}
                        target="_blank"
                        rel="noopener noreferrer"
                      >
                        Journal des modifications
                      </StyledLink>
                    );
                  })()}
                </StyledListItem>
              </StyledList>
            </Category>
            <Category>
              <Heading as={StyledStrong} isFirst>
                En collaboration avec
              </Heading>
              <StyledList>
                <StyledListItem>
                  <StyledLink
                    href={
                      "https://travail-emploi.gouv.fr/ministere/organisation/organisation-des-directions-et-services/article/organisation-de-la-direction-generale-du-travail-dgt"
                    }
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    La Direction Générale du Travail
                  </StyledLink>
                </StyledListItem>
                <StyledListItem>
                  <StyledLink
                    href={"https://fabrique.social.gouv.fr/"}
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    La fabrique des Ministères sociaux
                  </StyledLink>
                </StyledListItem>
                <StyledListItem>
                  <StyledLink
                    href={"https://beta.gouv.fr/"}
                    target="_blank"
                    rel="noopener noreferrer"
                  >
                    beta.gouv.fr
                  </StyledLink>
                </StyledListItem>
              </StyledList>
            </Category>
          </Links>
        </NavSection>
        <GovernmentSection>
          <StyledGovLink
            href={"https://travail-emploi.gouv.fr"}
            target="_blank"
            rel="noopener noreferrer"
          >
            travail-emploi.gouv.fr
          </StyledGovLink>
          <Separator aria-hidden>|</Separator>
          <StyledGovLink
            href={"https://www.service-public.fr"}
            target="_blank"
            rel="noopener noreferrer"
          >
            service-public.fr
          </StyledGovLink>
          <Separator aria-hidden>|</Separator>
          <StyledGovLink
            href={"https://www.legifrance.gouv.fr"}
            target="_blank"
            rel="noopener noreferrer"
          >
            legifrance.gouv.fr
          </StyledGovLink>
          <Separator aria-hidden>|</Separator>
          <StyledGovLink
            href={"https://www.data.gouv.fr"}
            target="_blank"
            rel="noopener noreferrer"
          >
            data.gouv.fr
          </StyledGovLink>
          <Separator aria-hidden>|</Separator>
          <StyledGovLink
            href={"https://www.gouvernement.fr"}
            target="_blank"
            rel="noopener noreferrer"
          >
            gouvernement.fr
          </StyledGovLink>
          <Separator aria-hidden>|</Separator>
          <StyledGovLink
            href={"https://www.cnil.fr/"}
            target="_blank"
            rel="noopener noreferrer"
          >
            cnil.fr
          </StyledGovLink>
        </GovernmentSection>
      </StyledFooter>
    </OverflowWrapper>
  );
};

export default Footer;

const { breakpoints, fonts, spacings } = theme;

const OverflowWrapper = styled.div`
  width: 100%;
  padding-top: ${spacings.larger};
  overflow-x: hidden;
`;

const StyledFooter = styled.footer`
  position: relative;
  z-index: 1;
  margin-top: ${spacings.larger};
  padding-top: ${spacings.base};
  background-color: ${({ theme }) => theme.bgSecondary};

  &:before {
    position: absolute;
    top: -27px;
    left: -50%;
    z-index: -1;
    width: 200%;
    height: 400px;
    background-color: ${({ theme }) => theme.bgSecondary};
    border-radius: 100%;
    content: "";
  }

  @media print {
    display: none;
  }
`;

const ServiceSection = styled(Section)`
  position: relative;
  padding-bottom: 7rem;
  text-align: center;
  @media (max-width: ${breakpoints.mobile}) {
    padding-top: 0;
    padding-bottom: ${spacings.larger};
  }
`;

const StyledContainer = styled(Container)`
  margin-bottom: ${spacings.medium};
`;

const StyledDirectionRightIcon = styled(DirectionRightIcon)`
  width: 1.5em;
  margin-left: ${spacings.base};
`;

const NavSection = styled(Section)`
  padding: 5rem 0;
  background-color: ${({ theme }) => theme.bgTertiary};
  @media (max-width: ${breakpoints.mobile}) {
    padding-bottom: ${spacings.larger};
  }
`;

const Links = styled(Container)`
  display: flex;
  justify-content: space-between;
  @media (max-width: ${breakpoints.mobile}) {
    flex-direction: column;
  }
`;

const Category = styled.div`
  & + & {
    padding-left: ${spacings.base};
    @media (max-width: ${breakpoints.mobile}) {
      padding-left: 0;
    }
  }
`;

const StyledList = styled(FlatList)`
  @media (max-width: ${breakpoints.mobile}) {
    text-align: center;
  }
`;

const StyledStrong = styled.strong`
  font-size: ${fonts.sizes.default};
  @media (max-width: ${breakpoints.mobile}) {
    margin-top: ${spacings.small};
    text-align: center;
  }
`;

const StyledListItem = styled.li`
  margin: ${spacings.tiny} 0;
`;

const StyledLink = styled.a.attrs((props) => ({
  ...(props.target === "_blank" && {
    "aria-label": `${props.children} (Nouvelle fenêtre)`,
  }),
}))`
  font-weight: normal;
  text-decoration: none;
  cursor: pointer;
`;

const GovernmentSection = styled.div`
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: center;
  background-color: ${({ theme }) => theme.bgTertiary};
`;

const StyledGovLink = styled(StyledLink)`
  margin: 0 ${spacings.small} ${spacings.small} ${spacings.small};
`;

const Separator = styled.span`
  margin-bottom: ${spacings.small};
  user-select: none;
  @media (max-width: ${breakpoints.tablet}) {
    display: none;
  }
`;
