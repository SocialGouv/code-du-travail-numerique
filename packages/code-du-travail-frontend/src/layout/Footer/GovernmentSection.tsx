import Link from "next/link";
import Image from "next/image";
import styled from "styled-components";
import { theme, Container } from "@socialgouv/cdtn-ui";
import { GITHUB_REPO, PACKAGE_VERSION } from "../../config";
import { ContactModal } from "../../common/ContactModal";
import { useRouter } from "next/router";
import { push as matopush } from "@socialgouv/matomo-next";

export const GouvernementSection = () => {
  const router = useRouter();
  const path = router.asPath;
  return (
    <StyledContainer>
      <StyledTopSection>
        <Image
          alt="le site du ministere du travail, de l'emploi et de l'insertion"
          src="/static/assets/img/ministere_logo.svg"
          width="170"
          height="141"
        />
        <StyledTopUl>
          <li>
            <Link href={"https://travail-emploi.gouv.fr"}>
              travail-emploi.gouv.fr
            </Link>
          </li>
          <li>
            <Link href={"https://www.service-public.fr"}>
              service-public.fr
            </Link>
          </li>
          <li>
            <Link href={"https://www.legifrance.gouv.fr"}>
              legifrance.gouv.fr
            </Link>
          </li>
          <li>
            <Link href={"https://www.data.gouv.fr"}>data.gouv.fr</Link>
          </li>
          <li>
            <Link href={"https://www.gouvernement.fr"}>gouvernement.fr</Link>
          </li>
        </StyledTopUl>
      </StyledTopSection>
      <SectionSeparator />
      <StyledBottomSection>
        <StyledBottomUl>
          <li>
            <Link href="/mentions-legales">Mentions légales</Link>
          </li>
          <li>
            <Link href="/accessibilite">Accessibilité&nbsp;: partiellement conforme</Link>
          </li>
          <li>
            <Link href="/politique-confidentialite">Politique de confidentialité</Link>
          </li>
          <li>
            <Link rel="noopener noreferrer" href={`${GITHUB_REPO}/tree/v${PACKAGE_VERSION}`}>Contribuer sur Github</Link>
          </li>
          <li>
            <ContactModal>
              {(openModal) => (
                <StyledContactLink
                  aria-label="Nous contacter"
                  onClick={() => {
                    matopush([
                      "trackEvent",
                      "contact",
                      "click_contact_cdtn_team",
                      path,
                    ]);
                    openModal();
                  }}
                >
                  Nous contacter
                </StyledContactLink>
              )}
            </ContactModal>
          </li>
        </StyledBottomUl>
      </StyledBottomSection>
    </StyledContainer>
  );
};

const { spacings, breakpoints } = theme;

const StyledContainer = styled(Container)`
  display: flex;
  flex-direction: column;
`;

const StyledTopSection = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding-bottom: ${spacings.medium};
  padding-top: ${spacings.medium};
  @media (max-width: ${breakpoints.tablet}) {
    padding-bottom: ${spacings.small};
    padding-top: ${spacings.small};
  }
`;

const StyledTopUl = styled.ul`
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  list-style: none;
  padding: 0;
  margin: 0;
  li {
    margin: 0 ${spacings.small};
    @media (max-width: ${breakpoints.tablet}) {
      margin: 0 ${spacings.xsmall};
    }
  }
  a {
    text-decoration: none;
    &:after {
      content: "" !important;
    }
  }
  @media (max-width: ${breakpoints.mobile}) {
    flex-direction: column;
  }
  flex-wrap: wrap;
`;

const StyledBottomSection = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: center;
  padding: ${spacings.medium} 0;
  @media (max-width: ${breakpoints.tablet}) {
    padding: ${spacings.small};
    margin-left: 0;
    justify-content: center;
  }
`;

const StyledBottomUl = styled(StyledTopUl)`
  a {
    color: #666;
    font-weight: 400;
    margin-right: ${spacings.small};
  }
`;

const SectionSeparator = styled.div`
  width: 100%;
  align-self: center;
  height: 1px;
  background-color: #666;
  margin: ${spacings.base} 0;
  @media (max-width: ${breakpoints.tablet}) {
    margin: ${spacings.small} 0;
  }
`;

const StyledContactLink = styled.a.attrs({
  role: "link",
  tabindex: "0"
})`
  cursor: pointer;
`;

