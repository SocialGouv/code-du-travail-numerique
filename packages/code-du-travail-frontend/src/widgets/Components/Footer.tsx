import Link from "next/link";
import styled from "styled-components";
import { theme } from "@socialgouv/cdtn-ui";
import { LogoLink } from "./LogoLink";

export const Footer = () => {
  return (
    <StyledFooter>
      <Link
        href="/politique-confidentialite"
        target="_blank"
        rel="noopener noreferrer"
      >
        Politique de confidentialit√©
      </Link>
      <LogoLink></LogoLink>
    </StyledFooter>
  );
};

const StyledFooter = styled.footer`
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: ${theme.spacings.base};
  @media print {
    display: none;
  }
`;
