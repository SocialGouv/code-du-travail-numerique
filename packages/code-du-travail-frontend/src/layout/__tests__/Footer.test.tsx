import { render } from "@testing-library/react";
import React from "react";

import Footer from "../Footer";

describe("<Footer />", () => {
  it("should render", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
