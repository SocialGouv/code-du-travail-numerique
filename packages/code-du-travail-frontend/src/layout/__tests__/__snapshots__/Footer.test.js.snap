// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
.c4 {
  max-width: 124rem;
  margin: 0 auto;
  padding: 0 2rem;
}

.c8 {
  max-width: 124rem;
  margin: 0 auto;
  padding: 0 2rem;
  padding: 0;
  max-width: 74rem;
}

.c2 {
  padding: 3.2rem 0;
  background-color: transparent;
  color: #3e486e;
}

.c7 {
  position: absolute;
  background-color: #7994d4;
  border-radius: 0.6rem;
  top: 0;
  left: 50%;
  width: 7rem;
  height: 0.4rem;
  -webkit-transform: translateX(-50%);
  -ms-transform: translateX(-50%);
  transform: translateX(-50%);
}

.c16 {
  position: relative;
  display: block;
  margin: 0 0 2rem 0;
  color: #2f3b6c;
  font-weight: 600;
  font-size: 1.8rem;
  font-family: "Open Sans",sans-serif;
  line-height: 1.625;
}

.c10 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: content-box;
  font-weight: 500;
  font-size: 1.6rem;
  line-height: 1.125em;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 0.6rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms linear, border-color 250ms linear,-webkit-transform 100ms linear;
  -webkit-transition: background-color 250ms linear, border-color 250ms linear,transform 100ms linear;
  transition: background-color 250ms linear, border-color 250ms linear,transform 100ms linear;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  height: 5.2rem;
  padding: 0 4.4rem;
  color: #fff;
  background: #7994d4;
  border-color: #7994d4;
  box-shadow: 0px 10px 20px rgba(121,148,212,0.26),0px 5px 7px rgba(121,148,212,0.35);
  opacity: 1;
  will-change: transform;
}

.c10:link,
.c10:visited {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #fff;
}

.c10:not([disabled]):hover,
.c10:not([disabled]):active,
.c10:not([disabled]):focus {
  opacity: 1;
  -webkit-transform: translateY(-2px);
  -ms-transform: translateY(-2px);
  transform: translateY(-2px);
  background: #a0b3e0;
  border-color: #a0b3e0;
}

.c10[disabled] {
  background-color: #e4e8ef;
  border-color: #e4e8ef;
  color: #9298af;
  box-shadow: none;
  cursor: not-allowed;
}

.c20 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-sizing: content-box;
  font-weight: 500;
  font-size: 1.6rem;
  line-height: 1.125em;
  text-align: center;
  vertical-align: middle;
  border: 1px solid;
  border-radius: 0.6rem;
  cursor: pointer;
  -webkit-transition: background-color 250ms linear, border-color 250ms linear,-webkit-transform 100ms linear;
  -webkit-transition: background-color 250ms linear, border-color 250ms linear,transform 100ms linear;
  transition: background-color 250ms linear, border-color 250ms linear,transform 100ms linear;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  padding: 0;
  color: #3e486e;
  font-weight: normal;
  font-size: 1.6rem;
  line-height: 1.625;
  vertical-align: baseline;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: left;
  background: none;
  border: none;
  border-radius: 0;
  overflow: visible;
  -webkit-transition: color 250ms linear, text-decoration 250ms linear;
  transition: color 250ms linear, text-decoration 250ms linear;
}

.c20:focus,
.c20:hover,
.c20:active {
  color: #f66663;
}

.c17 {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

.c5 {
  margin-top: 0;
  margin-bottom: 2rem;
  margin-left: auto;
  text-align: center;
}

.c6 {
  position: relative;
  margin: 0;
  color: #2f3b6c;
  font-weight: normal;
  font-size: 2.8rem;
  font-family: "Merriweather",serif;
  line-height: 1.25;
  padding-top: 1.6rem;
  text-align: center;
}

.c0 {
  width: 100%;
  padding-top: 4rem;
  overflow-x: hidden;
}

.c1 {
  position: relative;
  z-index: 1;
  margin-top: 4rem;
  padding-top: 1.6rem;
  background-color: #f2f5fa;
}

.c1:before {
  position: absolute;
  top: -27px;
  left: -50%;
  z-index: -1;
  width: 200%;
  height: 400px;
  background-color: #f2f5fa;
  border-radius: 100%;
  content: "";
}

.c3 {
  position: relative;
  padding-bottom: 7rem;
  text-align: center;
}

.c9 {
  margin-bottom: 2rem;
}

.c11 {
  width: 1.5em;
  margin-left: 1.6rem;
}

.c12 {
  padding: 5rem 0;
  background-color: #e4e8ef;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c14 + .c14 {
  padding-left: 1.6rem;
}

.c15 {
  font-size: 1.6rem;
}

.c19 {
  font-weight: normal;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  padding: 0.4rem 0;
  display: inline-block;
}

.c21 {
  padding: 0.4rem 0;
}

.c22 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background-color: #f2f5fa;
  list-style-type: none;
  margin: 0;
  padding-bottom: 1.6rem;
}

.c23 {
  margin: 0 1rem 1rem 1rem;
}

.c24 {
  margin-bottom: 1rem;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

@media (max-width:600px) {
  .c4 {
    padding: 0 1rem;
  }
}

@media print {
  .c4 {
    max-width: 100%;
    padding: 0;
  }
}

@media (max-width:600px) {
  .c8 {
    padding: 0;
  }
}

@media print {
  .c8 {
    max-width: 100%;
    padding: 0;
  }
}

@media (max-width:600px) {
  .c2 {
    padding: 2.4rem 0;
  }
}

@media (max-width:600px) {
  .c16 {
    font-size: 1.6rem;
  }
}

@media (max-width:600px) {
  .c10 {
    font-size: 1.4rem;
  }
}

@media print {
  .c10 {
    display: none;
  }
}

@media (max-width:600px) {
  .c10 {
    padding: 0 3rem;
  }
}

@media (max-width:600px) {
  .c20 {
    font-size: 1.4rem;
  }
}

@media print {
  .c20 {
    display: none;
  }
}

@media (max-width:600px) {
  .c5 {
    margin-bottom: 1rem;
    margin-left: auto;
  }
}

@media (max-width:600px) {
  .c6 {
    font-size: 2.2rem;
  }
}

@media print {
  .c1 {
    display: none;
  }
}

@media (max-width:600px) {
  .c3 {
    padding-top: 0;
    padding-bottom: 4rem;
  }
}

@media (max-width:600px) {
  .c12 {
    padding-bottom: 4rem;
  }
}

@media (max-width:600px) {
  .c13 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:600px) {
  .c14 + .c14 {
    padding-left: 0;
  }
}

@media (max-width:600px) {
  .c18 {
    text-align: center;
  }
}

@media (max-width:600px) {
  .c15 {
    margin-top: 1rem;
    text-align: center;
  }
}

@media (max-width:980px) {
  .c24 {
    display: none;
  }
}

<div>
  <div
    class="c0"
  >
    <footer
      class="c1"
    >
      <div
        class="c2 c3"
      >
        <div
          class="c4"
        >
          <div
            aria-level="2"
            class="c5"
            role="heading"
          >
            <p
              class="c6"
            >
              <span
                class="c7"
              />
              Besoin de plus d’informations ?
            </p>
          </div>
          <p
            class="c8 c9"
          >
            <p>
              Les services du ministère du Travail en région informent, conseillent et orientent les salariés et les employeurs du secteur privé sur leurs questions en droit du travail.
            </p>
          </div>
          <button
            class="c10"
          >
            Contacter nos services en région
            <svg
              aria-hidden="true"
              class="c11"
              fill="none"
              viewBox="0 0 28 15"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M27.875 7.153l-6-6.284a.526.526 0 00-.711-.023.496.496 0 00-.024.688l5.204 5.468H.5c-.276 0-.5.217-.5.484s.224.483.5.483h25.844l-5.204 5.469a.493.493 0 00.024.687.52.52 0 00.711-.023l6-6.284a.506.506 0 000-.665z"
                fill="currentColor"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        class="c2 c12"
      >
        <div
          class="c4 c13"
        >
          <div
            class="c14 "
          >
            <strong
              aria-level="2"
              class="c15 c16"
              role="heading"
            >
              Code du travail numérique
            </strong>
            <ul
              class="c17 c18"
            >
              <li>
                <a
                  class="c19"
                  href="/droit-du-travail"
                >
                  Le droit du travail
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/glossaire"
                >
                  Glossaire
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/a-propos"
                >
                  À propos
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/mentions-legales"
                >
                  Mentions légales
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/accessibilite"
                >
                  Accessibilité : partiellement conforme
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/politique-confidentialite"
                >
                  Politique de confidentialité
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/stats"
                >
                  Statistiques d’utilisation
                </a>
              </li>
              <li>
                <a
                  class="c19"
                  href="/integration"
                >
                  Intégrer le Code du travail numérique
                </a>
              </li>
            </ul>
          </div>
          <div
            class="c14 "
          >
            <strong
              aria-level="2"
              class="c15 c16"
              role="heading"
            >
              Aidez-nous à améliorer cet outil
            </strong>
            <ul
              class="c17 c18"
            >
              <li>
                <button
                  class="c20 c21"
                >
                  Contact
                </button>
              </li>
              <li>
                <a
                  aria-label="Contribuer sur Github (Nouvelle fenêtre)"
                  class="c19"
                  href="https://github.com/SocialGouv/code-du-travail-numerique/tree/vX.Y.Z"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  Contribuer sur Github
                </a>
              </li>
              <li>
                <a
                  aria-label="Journal des modifications (Nouvelle fenêtre)"
                  class="c19"
                  href="https://github.com/SocialGouv/code-du-travail-numerique/releases/tag/vX.Y.Z"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  Journal des modifications
                </a>
              </li>
            </ul>
          </div>
          <div
            class="c14 "
          >
            <strong
              aria-level="2"
              class="c15 c16"
              role="heading"
            >
              En collaboration avec
            </strong>
            <ul
              class="c17 c18"
            >
              <li>
                <a
                  aria-label="La Direction Générale du Travail (Nouvelle fenêtre)"
                  class="c19"
                  href="https://travail-emploi.gouv.fr/ministere/organisation/organisation-des-directions-et-services/article/organisation-de-la-direction-generale-du-travail-dgt"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  La Direction Générale du Travail
                </a>
              </li>
              <li>
                <a
                  aria-label="La fabrique des Ministères sociaux (Nouvelle fenêtre)"
                  class="c19"
                  href="https://fabrique.social.gouv.fr/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  La fabrique des Ministères sociaux
                </a>
              </li>
              <li>
                <a
                  aria-label="beta.gouv.fr (Nouvelle fenêtre)"
                  class="c19"
                  href="https://beta.gouv.fr/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  beta.gouv.fr
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <ul
        class="c22"
      >
        <li>
          <a
            aria-label="travail-emploi.gouv.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://travail-emploi.gouv.fr"
            rel="noopener noreferrer"
            target="_blank"
          >
            travail-emploi.gouv.fr
          </a>
          <span
            aria-hidden="true"
            class="c24"
          >
            |
          </span>
        </li>
        <li>
          <a
            aria-label="service-public.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://www.service-public.fr"
            rel="noopener noreferrer"
            target="_blank"
          >
            service-public.fr
          </a>
          <span
            aria-hidden="true"
            class="c24"
          >
            |
          </span>
        </li>
        <li>
          <a
            aria-label="legifrance.gouv.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://www.legifrance.gouv.fr"
            rel="noopener noreferrer"
            target="_blank"
          >
            legifrance.gouv.fr
          </a>
          <span
            aria-hidden="true"
            class="c24"
          >
            |
          </span>
        </li>
        <li>
          <a
            aria-label="data.gouv.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://www.data.gouv.fr"
            rel="noopener noreferrer"
            target="_blank"
          >
            data.gouv.fr
          </a>
          <span
            aria-hidden="true"
            class="c24"
          >
            |
          </span>
        </li>
        <li>
          <a
            aria-label="gouvernement.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://www.gouvernement.fr"
            rel="noopener noreferrer"
            target="_blank"
          >
            gouvernement.fr
          </a>
          <span
            aria-hidden="true"
            class="c24"
          >
            |
          </span>
        </li>
        <li>
          <a
            aria-label="cnil.fr (Nouvelle fenêtre)"
            class="c19 c23"
            href="https://www.cnil.fr/"
            rel="noopener noreferrer"
            target="_blank"
          >
            cnil.fr
          </a>
        </li>
      </ul>
    </footer>
  </div>
</div>
`;
